NetCoreDemos/ViewModels/GalleryViewModel.cs

```diff
@@ -3,7 +3,12 @@
 using System.Linq;
 
 namespace DevExtreme.NETCore.Demos.ViewModels {
+    public class ImageItem {
+        public string imageSrc { get; set; }
```

Comment:
Shouldn't it start with a capital letter?

Reply:
With a capital letter it does not work.